HIP/hip-1215.md

```diff
@@ -90,7 +90,7 @@ schedule and an `int64` that is the protobuf ordinal of enum value `"SCHEDULE_EX
 If the `scheduleCall()` variants succeed, they return the address of the newly created
 scheduled transaction and `22` as the status code of `SUCCESS`. A contract may attempt to
 delete an already scheduled transaction with the `deleteSchedule(address)`, receiving `22`
-iff the delete succeeds. A contract or EOA may also attempt to delete a scheduled transaction
```

Comment:
nit: to my ears, the way this was phrased originally sound correct but not sure.